configure

```diff
@@ -1517,6 +1517,7 @@ config = {
   'BUILDTYPE': 'Debug' if options.debug else 'Release',
   'PYTHON': sys.executable,
   'NODE_TARGET_TYPE': variables['node_target_type'],
+  'HAS_CRYPTO': variables['node_use_openssl'],
```

Comment:
I'd call this `NODE_USE_OPENSSL` for consistency with `NODE_TARGET_TYPE` (and also `grep -i` greppability.)

Reply:
I'll change this as well. Thanks